theano/sandbox/rng_mrg.py

```diff
@@ -1518,6 +1521,7 @@ def multinomial_wo_replacement(self, size=None, n=1, pvals=None,
         warnings.warn('MRG_RandomStreams.multinomial_wo_replacement() is '
                       'deprecated and will be removed in the next release of '
                       'Theano. Please use MRG_RandomStreams.choice() instead.')
+        assert size is None
         return self.choice(size=n, a=None, replace=False, p=pvals,
                            dtype=dtype, nstreams=nstreams, ndim=ndim)
```

Comment:
a isn't an input of multinomial_wo_replacement. I don't see an input of multinomial_wo_replacement that is ignored except size that we have the assert on. So I don't see a problem here.